tests/unit/BaseTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * This class caters for the different PHPUnit versions between production 
+ * and development.
+ * To execute unit tests especially during development set an environment
+ * variable TEST_ENV=<environment> either as 'dev' or 'prod' :
+ * 
+ *   export TEST_ENV=<environment>
+ *   php codecept.phar run unit --debug
+ */
+$test_env = null !== getenv('TEST_ENV') ? getenv('TEST_ENV') : 'prod';
+
+if ($test_env !== 'dev') {
+	// php 5.6, production environment
+	class BaseTestCase extends \PHPUnit_Framework_TestCase {}
+} else {
+	// php 7+, dev environment
+	class BaseTestCase extends PHPunit\Framework\TestCase {}
```

Comment:
This assumes that PHP 7+ is used for developing, which may not be always the case (for example if an engineer uses another docker-compose file with an older PHP version). I wonder if we can try / catch the error if the first TestCase class doesn't exist, then load the alternative one.

Reply:
Bravo @Diontsoumas, excellent idea! In fact PHP has the function class_exists() just for that ;)